AppCenterCrashes/AppCenterCrashes/MSCrashes.mm

```diff
@@ -847,7 +850,7 @@ - (void)sendErrorAttachments:(NSArray<MSErrorAttachmentLog *> *)errorAttachments
       MSLogError([MSCrashes logTag], @"Not all required fields are present in MSErrorAttachmentLog.");
       continue;
     }
-    [self.channelUnit enqueueItem:attachment];
+    [self.channelUnit enqueueItem:attachment flags:MSFlagsNone];
```

Comment:
Should it be `Normal`?

Reply:
None needs to map to default values for each flags as discussed.

Reply:
Oh wait crashes should be critical actually.